test/downstream/analysis_points.jl

```diff
@@ -438,3 +438,52 @@ end
     matrices, _ = get_sensitivity(sys, sys.ap)
     @test matrices == matrices_normal
 end
+
+@testset "Ignored analysis points only affect relevant connection sets" begin
+    m1 = 1
+    m2 = 1
+    k = 1000 # Spring stiffness
+    c = 10   # Damping coefficient
+
+    @named inertia1 = Inertia(; J = m1, phi = 0, w = 0)
+    @named inertia2 = Inertia(; J = m2, phi = 0, w = 0)
+
+    @named spring = Spring(; c = k)
+    @named damper = Damper(; d = c)
+
+    @named torque = Torque(use_support = false)
+
+    function SystemModel(u = nothing; name = :model)
+        eqs = [connect(torque.flange, inertia1.flange_a)
+               connect(inertia1.flange_b, spring.flange_a, damper.flange_a)
+               connect(inertia2.flange_a, spring.flange_b, damper.flange_b)]
+        if u !== nothing
+            push!(eqs, connect(torque.tau, u.output))
+            return @named model = ODESystem(
+                eqs, t; systems = [torque, inertia1, inertia2, spring, damper, u])
+        end
+        ODESystem(eqs, t; systems = [torque, inertia1, inertia2, spring, damper], name)
+    end
+
+    @named r = Step(start_time = 1)
+    @named pid = LimPID(k = 400, Ti = 0.5, Td = 1, u_max = 350)
+    @named filt = SecondOrder(d = 0.9, w = 10, x = 0, xd = 0)
+    @named sensor = AngleSensor()
+    @named add = Add() # To add the feedback and feedforward control signals
+    model = SystemModel()
+    @named inverse_model = SystemModel()
+    @named inverse_sensor = AngleSensor()
+    connections = [connect(r.output, :r, filt.input) # Name connection r to form an analysis point
+                   connect(inverse_model.inertia1.flange_b, inverse_sensor.flange) # Attach the inverse sensor to the inverse model
+                   connect(filt.output, pid.reference, inverse_sensor.phi) # the filtered reference now goes to both the PID controller and the inverse model input
+                   connect(inverse_model.torque.tau, add.input1)
+                   connect(pid.ctr_output, add.input2)
+                   connect(add.output, :u, model.torque.tau) # Name connection u to form an analysis point
+                   connect(model.inertia1.flange_b, sensor.flange)
+                   connect(sensor.phi, :y, pid.measurement)]
+    closed_loop = ODESystem(connections, t,
+        systems = [model, inverse_model, pid, filt, sensor, inverse_sensor, r, add],
+        name = :closed_loop)
+    # just ensure the system simplifies
+    Blocks.get_sensitivity(closed_loop, :y)
```

Comment:
```suggestion
    get_sensitivity(closed_loop, :y)
```
`get_sensitivity` is now in MTK and not in Blocks, right?

Should there be some test that the sensitivity function is correct?

Reply:
Something like this perhaps. If you are confident that the result is the same as that in the last successful docs build from where the example came, you can set  `reference_fr` to the computed `fr` in this PR, so that we have that as a test for future regressions
```julia
mats, _ = Blocks.get_sensitivity(closed_loop, :y)
S = CS.ss(mats...)
fr = CS.freqrespv(S, [0.01, 1, 100])
@test isapprox(fr, reference_fr)
```

Reply:
You can run this code and verify that the `S` function looks the same
![image](https://github.com/user-attachments/assets/97d4431a-3168-4387-a486-cbb72415c8d5)